api/v1alpha1/usersignup_types.go

```diff
@@ -196,6 +196,48 @@ type UserSignupSpec struct {
 	// a new IdP provider client, and contains the user's "original-sub" claim
 	// +optional
 	OriginalSub string `json:"originalSub,omitempty"`
+
+	// IdentityClaims contains as-is claim values extracted from the user's access token
+	IdentityClaims IdentityClaimsEmbedded `json:"identityClaims,omitempty"`
+}
+
+// IdentityClaimsEmbedded is used to define a set of SSO claim values that we are interested in storing
+// +k8s:openapi-gen=true
+type IdentityClaimsEmbedded struct {
+
+	// PropagatedClaims
```

Comment:
Not sure if we need it here...
```suggestion
	// PropagatedClaims
	// +optional
```

Reply:
fixed